Basic-Video-Chat/.gitignore

```diff
@@ -0,0 +1,16 @@
+# intellij
+*.iml
+
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+app/build
+
+.settings/
+app/jniLibs/
+app/libs/opentok-android-sdk.jar
```

Comment:
What if someone wants to use the .jar file? I know this is not common, but if they want to, it should be ok. In that case we shouldn't track the file.